runtime/lua/vim/_defaults.lua

```diff
@@ -534,14 +534,48 @@ do
     end,
   })
 
+  local nvim_terminal_shell_integration_ns =
+    vim.api.nvim_create_namespace('nvim.terminal.shell_integration')
```

Comment:
```suggestion
  local nvim_terminal_prompt_ns =
    vim.api.nvim_create_namespace('nvim.terminal.prompt')
```

Not sure - but "shell-integration" sounds somewhat generic and if you used the same namespace to set other extmarks, then it would break `jump_to_prompt`

Reply:
Changed in 1a694c9b2104